File: Data Structures & Algorithms/duplicate-integer/submission-1.cs
public class Solution {
    public bool hasDuplicate(int[] nums) {
        HashSet<int> discovered = new HashSet<int>();
        for (int i = 0; i < nums.Length; i++) {
            if (discovered.Contains(nums[i])) {
                return true;
            }
            else {
                discovered.Add(nums[i]);
            }
        }
        return false;
    }
}

File: Data Structures & Algorithms/remove-element/submission-1.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        jump = 0
        for i in range(0, len(nums)):
            if nums[i] == val:
                jump+=1
            else:
                nums[i - jump] = nums[i]
        return len(nums) - jump